connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -302,12 +302,27 @@ export async function confluenceCheckAndUpsertPageActivity({
       baseUrl: confluenceConfig.url,
       suffix: page._links.tinyui,
     });
+
+    // We log the number of labels to help define the importance of labels in the future.
+    if (page.labels.results.length > 0) {
+      localLogger.info(
+        { labelsCount: page.labels.results.length },
```

Comment:
Maybe add a conditional on whether we will strip or not ?

Reply:
I want the count so I can check in DD the P90/P95.